gdal-sys/build.rs

```diff
@@ -0,0 +1,50 @@
+
+
+fn main() {
+
+    let link_type = "dylib";
+    let lib_name = "gdal";
+    
+    #[cfg(target_family="windows")]

```

Comment:
https://doc.rust-lang.org/reference.html#conditional-compilation

You should be able to make this just `#[cfg(windows)]`